### PR TITLE
Fix exception when using with jellyfin 10.4.0

### LINKED ISSUE #25: 500 on Authorize device - 10.4.0 nightly 2019-09-19
Jellyfin: 10.4.0 nightly 2019-09-19
Trakt plugin: 5.0.0.0
```
[2019-09-19 20:04:42.489 +02:00] [INF] DeviceAuthorization request received
[2019-09-19 20:04:42.491 +02:00] [ERR] Error processing request
System.MissingMethodException: Method not found: 'Void MediaBrowser.Common.Net.HttpRequestOptions.set_LogRequest(Boolean)'.
   at Trakt.Api.TraktApi.GetHttpRequestOptions()
   at Trakt.Api.TraktApi.PostToTrakt(String url, Object data, CancellationToken cancellationToken, TraktUser traktUser)
```

### PR DESCRIPTION
**Pull request by Bond-009**
Fixes #25

### CODE REVIEW

**Comment by joshuaboniface** [comment]
Version bump please :+1:

File: build.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: "jellyfin-plugin-trakt"
 guid: "4fe3201e-d6ae-4f2e-8917-e12bda571281"
-version: "5" # Please increment with each pull request
+version: "6" # Please increment with each pull request
 jellyfin_version: "10.3.6" # The earliest binary-compatible version
```
Comment (joshuaboniface) [inline build.yaml]: Do we know if this breaks backwards compat?

Comment (Bond-009) [reply build.yaml]: It doesn't.

Comment (joshuaboniface) [reply build.yaml]: Solid :+1:

**Pull request merged**